src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -187,21 +212,16 @@ void GraphingCalculator::OnDataRequested(DataTransferManager ^ sender, DataReque
 
         // Set the thumbnail image (in case the share target can't handle HTML)
         requestData->Properties->Thumbnail = bitmapStream;
-
-        // And the bitmap (in case the share target can't handle HTML)
-        requestData->SetBitmap(bitmapStream);
```

Comment:
I wouldn't remove this, if they are sharing to outlook or other possible non html target the thumbnail / bitmap can end up being rendered differently.

Reply:
I noticed that sharing to outlook doesn't seem to bring over any data, including this bitmap, seems like an issue on the receiving end? While sharing via mail it includes the html **and** the image as an attachment which seems noisy.